src/Nordea/Components/FeatureBox.elm

```diff
@@ -0,0 +1,172 @@
+module Nordea.Components.FeatureBox exposing
+    ( FeatureBox
+    , init
+    , view
+    , withButton
+    , withIcon
+    )
+
+import Css
+    exposing
+        ( Style
+        , alignItems
+        , auto
+        , backgroundColor
+        , bottom
+        , center
+        , column
+        , displayFlex
+        , fixed
+        , flexDirection
+        , height
+        , important
+        , int
+        , justifyContent
+        , left
+        , marginBottom
+        , marginLeft
+        , marginRight
+        , marginTop
+        , minWidth
+        , none
+        , outline
+        , overflow
+        , padding
+        , padding4
+        , pct
+        , position
+        , pseudoClass
+        , rem
+        , textAlign
+        , top
+        , width
+        , zIndex
+        )
+import Css.Global as Css exposing (children, everything)
+import Html.Styled
+    exposing
+        ( Html
+        , div
+        , h2
+        , p
+        , text
+        )
+import Html.Styled.Attributes exposing (css)
+import Html.Styled.Events exposing (onClick)
+import Nordea.Components.Button as Button
+import Nordea.Components.Text as Text exposing (..)
+import Nordea.Html as Html exposing (showIf)
+import Nordea.Resources.Colors as Colors
+import Nordea.Resources.Icons as Icons
+import Nordea.Themes as Themes
+import Svg.Styled as Svg exposing (Svg)
+
+
+type alias Config msg =
+    { showModal : Bool

```

Comment:
we now have another component that is called modal - to not confuse them, should we call this showFeatureBox?

Reply:
Yes changed to showFeatureBox 👍